pkg/install/deploystorage.go

```diff
@@ -228,6 +228,22 @@ func (i *Installer) deployStorageTemplate(ctx context.Context, installConfig *in
 							},
 							NotActions: &[]string{
 								"Microsoft.Network/networkSecurityGroups/join/action",
+								"Microsoft.Compute/disks/read",
+								"Microsoft.Compute/disks/write",
+								"Microsoft.Compute/disks/beginGetAccess/action",
+								"Microsoft.Compute/disks/endGetAccess/action",
+								"Microsoft.Compute/snapshots/read",
+								"Microsoft.Compute/snapshots/write",
+								"Microsoft.Compute/snapshots/beginGetAccess/action",
+								"Microsoft.Compute/snapshots/endGetAccess/action",
+								"Microsoft.Compute/diskAccesses/read",
+								"Microsoft.Compute/diskAccesses/write",
+								"Microsoft.Compute/diskAccesses/privateEndpointConnectionsApproval/action",
+								"Microsoft.Compute/diskAccesses/privateEndpointConnectionProxies/read",
+								"Microsoft.Compute/diskAccesses/privateEndpointConnectionProxies/write",
+								"Microsoft.Compute/diskAccesses/privateEndpointConnectionProxies/validate/action",
```

Comment:
Weird that the split setting shows this line not formatted correctly unlike the unified setting.